.github/ISSUE_TEMPLATE/release.md

```diff
@@ -68,35 +70,38 @@ This will not be so crucial with release branches, but is nonetheless sensible a
 ```[tasklist]
 #### Technical tasks
 - [ ] Temporarily remove branch protection before pushing the release branches/tags
-- [ ] Create release branches for docker-images (see stackable-utils for script to create branches)
+- [ ] Create release branch for docker-images (see stackable-utils for script to create branches)
 - [ ] Create release tag(s) for docker-images (see stackable-utils for scripts to create tags)
 - [ ] Create release branches for operators (see stackable-utils for script to create branches)
 - [ ] Create release tag(s) for operators (see stackable-utils for scripts to create tags)
 - [ ] Create release tag for stackable-cockpit (optional, highly experimental, requires manual tag creation)
 - [ ] Update release version in changelogs on main branches (see stackable-utils for script to do this)
+- [ ] Create release branch for demos (see stackable-utils for script to create branches)
 - [ ] Generate CRD docs [website](https://crds.stackable.tech/) for the new release by following these [instructions](https://github.com/stackabletech/crddocs)
-- [ ] Check (selected) integration tests on the new release branches
-- [ ] Run/Check getting started scripts, ensure image pulls for the release works
-- [ ] Run/Check demos with dev release and main branch and create draft PR for release-related changes
 - [ ] Test `stackablectl` with locally updated (to new release number) `releases.yaml`
 - [ ] Update `release.yaml` in https://github.com/stackabletech/release/blob/main/releases.yaml
-- [ ] Check that an upgrade can be performed on an existing cluster without data loss.
+- [ ] ARM demo and integration testing of some kind
+- [ ] [Check and update getting-started script](https://github.com/stackabletech/issues/issues/new?template=TODO-getting-started-scripts.md)
+- [ ] [Test demos for the new release](https://github.com/stackabletech/demos/issues/new?template=TODO-release-testing.md)
+- [ ] Check that an upgrade can be performed on an existing cluster without data loss (cycling demo)
 - [ ] Run all of the test suites
+- [ ] OpenShift testing on Rosa <!-- where is that at? -->
 ```
 
+> [!TIP]
+> Name the release-notes branch `docs/release-notes-XX.XX.X` so that the link below takes you directly to the [Pull Request template][docs-pr-template].
```

Comment:
suggestion: Unify the version placeholder.

```suggestion
> Name the release-notes branch `docs/release-notes-YY.M.X` so that the link below takes you directly to the [Pull Request template][docs-pr-template].
```